### PR TITLE
Onnx opset11 rebased

### PR DESCRIPTION
**Pull request by ericwu155214**
Hi,

here is update:

caffe convertor tested under opset11
keras convertor upgraded to opset11
tflite convertor upgraded to opset11

the old branch just broken in accident,
here is the new one

please check it,
thanks!

### CODE REVIEW

**Review by MrWhoami — CHANGES_REQUESTED** [review]

**Comment by ericwu155214** [comment]
how about this, I made a new function for creating scarlar for this case.

File: keras-onnx/onnx_keras/aact_layers.py [hunk 1]
```diff
@@ -41,24 +44,43 @@ def generate(self):
...
         inputs=self.inputs,
         outputs=self.outputs,
         name=self.name,
-        min=float(self.layer.threshold)
       )
+      node_list.append(threshold_value_node[0])
+      node_list.append(node)
     else:
+      threshold = np.array(self.layer.threshold)
+      max_value = np.array(self.layer.max_value)
```
[the remark this answers is not in the document]
Comment (ericwu155214) [reply keras-onnx/onnx_keras/aact_layers.py:65]: I push a new commit for this, please check it.

File: keras-onnx/onnx_keras/frontend.py [hunk 2]
```diff
@@ -221,7 +221,6 @@ def convertToOnnx(self, optimize=0, input_shape=None):
       )
     # Create the model (ModelProto)
     self.omodel = O.helper.make_model(graph_def, producer_name='Kneron')
-    self.omodel.opset_import[0].version = 9
```
Comment (MrWhoami) [inline keras-onnx/onnx_keras/frontend.py:224]: We may set it to 11 instead of removing it.

**Review by MrWhoami — CHANGES_REQUESTED** [review]

File: keras-onnx/onnx_keras/frontend.py (hunk 2 above)
Comment (ericwu155214) [reply keras-onnx/onnx_keras/frontend.py:224]: Sure, let me add it back

**Comment by ericwu155214** [comment]
how about this, I add it back.

**Review by MrWhoami — APPROVED** [review]

**Pull request merged**